ucm2/sof/product_configs/AAEON/UPX-TGL01.conf

```diff
@@ -0,0 +1,20 @@
+# Add customization of IIR, FIR, and DRC
+# Note: Use the correct IPC version for the blobs path
+
+# Example set all processing to bypass
+Define.SpeakerIirBlob "/usr/share/alsa/ucm2/sof/ipc4/eq_iir/pass.blob"
+Define.SpeakerFirBlob "/usr/share/alsa/ucm2/sof/ipc4/eq_fir/pass.blob"
+Define.SpeakerDrcBlob "/usr/share/alsa/ucm2/sof/ipc4/drc/passthrough.blob"
+
+# Headphone is by default passthrough in HiFi-analog-sof.conf
+#Define.HeadphoneIirBlob "/usr/share/alsa/ucm2/sof/ipc4/eq_iir/pass.blob"
+#Define.HeadphoneFirBlob "/usr/share/alsa/ucm2/sof/ipc4/eq_fir/pass.blob"
+#Define.HeadphoneDrcBlob "/usr/share/alsa/ucm2/sof/ipc4/drc/passthrough.blob"
+
+# Example customize all
+#Define.SpeakerIirBlob "/usr/share/alsa/ucm2/sof/ipc4/eq_iir/speaker_aaeon_upx-tgl01.blob"
+#Define.SpeakerFirBlob "/usr/share/alsa/ucm2/sof/ipc4/eq_fir/speaker_aaeon_upx-tgl01.blob"
+#Define.SpeakerDrcBlob "/usr/share/alsa/ucm2/sof/ipc4/drc/speaker_aaeon_upx-tgl01.blob"
+#Define.HeadphoneIirBlob "/usr/share/alsa/ucm2/sof/ipc4/eq_iir/headphone_aaeon_upx-tgl01.blob"
+#Define.HeadphoneFirBlob "/usr/share/alsa/ucm2/sof/ipc4/eq_fir/headphone_aaeon_upx-tgl01.blob"
+#Define.HeadphoneDrcBlob "/usr/share/alsa/ucm2/sof/ipc4/drc/headphone_aaeon_upx-tgl01.blob"
```

Comment:
Sorry, we should not rely on people editing files. That's a big no-no for files updated by a distro.

Reply:
I added those as comments for what a vendor can customize, it might not be clear from the HiFi-analog-sof.conf. Your idea for user's customization is good so I will try to add such possibility. It will be these same UCM variables.

Reply:
Also I'm not sure UPX is a good example, not sure it can ever have UCM in speaker mode. But since Intel is not selling consumer devices, not even NuC any more, so I though UPX that most of us developers use, might be the best harmless example.